mockups/pain-staged.html

```
@@ -0,0 +1,447 @@
+    <a href="worksheet-b.html">Reference: Worksheet B</a>
+  </nav>
+</div>
+
+<div class="page">
+    <nav class="page-tabs">
+    <a class="current" href="#">New entry</a>
+    <a href="#">History</a>
+  </nav>
+  <header class="m-head">
+    <div class="m-titlerow">
+      <button class="m-burger" aria-label="Open navigation" aria-haspopup="dialog"><svg width="18" height="18" viewBox="0 0 18 18" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"><path d="M2 4.5h14M2 9h14M2 13.5h14"/></svg></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)